package.json

```diff
@@ -72,26 +70,24 @@
     "prettier": "2.7.1",
     "pretty-quick": "^4.0.0",
     "prop-types": "^15.6.0",
-    "raw-loader": "^1.0.0",
     "react": "^16.14.0",
     "react-dom": "^16.14.0",
     "redbox-react": "^1.3.6",
     "resolve-url-loader": "^5.0.0",
-    "sass": "<1.80",
-    "sass-loader": "^10",
+    "sass": "^1.81.0",
+    "sass-loader": "^16.0.3",
     "semver": "^5.0.0",
-    "style-loader": "^0.23.1",
+    "style-loader": "^4.0.0",
     "svg-sprite-loader": "^6.0.11",
-    "terriajs": "8.7.11",
-    "terriajs-cesium": "8.0.1",
+    "terriajs": "terriajs/terriajs#webpack5",
+    "terriajs-cesium": "8.0.2",
     "terriajs-plugin-api": "0.0.1-alpha.16",
     "terriajs-plugin-sample": "0.0.1-alpha.7",
     "typescript": "~5.2.0",
     "urijs": "^1.18.12",
-    "url-loader": "^1.1.2",
-    "webpack": "~4.47.0",
-    "webpack-cli": "^4.10.0",
-    "webpack-dev-server": "^4.15.2",
+    "webpack": "^5.96.1",
```

Comment:
There is a resolution for terser-webpack-plugin that should be removed in this package.json as well.